modules/test/playwright/pages/users-admin-web/EditUserPage.ts

```diff
@@ -85,5 +91,19 @@ export class EditUserPage {
 				strictEqual
 			);
 		};
+		this.screenNameInput = page.getByLabel("Screen Name");
+		this.emailAddressInput = page.getByLabel("Email Address");
+		this.saveButton = page.getByRole('button', {name: 'Save'});
+		this.passwordConfirmationFrame = page.frameLocator('iframe[title="Confirm Password"]')
+		this.yourPasswordInput = this.passwordConfirmationFrame.getByLabel('Your Password')
+		this.confirmButton = this.passwordConfirmationFrame.getByRole('button', {name: 'Confirm'});
```

Comment:
same here, sort them... only the latest two can remain at the end because they are using not page but passwordConfirmationFrame (but between them they should be sorted)